client/my-sites/people/main.jsx

```diff
@@ -53,6 +53,15 @@ export default React.createClass( {
 		}
 	},
 
+	renderPeopleSectionNav: function( site ) {
+		const commonProps = omit( this.props, [ 'sites' ] );
+		if ( this.props.filter === 'team' ) {
+			return <PeopleSectionNav { ...commonProps } site={ site } baseUrl={ `/people/team/${this.props.sites.selected}` } />;
```

Comment:
It is super hard to track what effect `baseUrl` has exactly. It’s neither in the `PeopleSectionNav`’s docs, nor in its source-code.